tox.ini

```diff
@@ -4,6 +4,11 @@ skip_missing_interpreters = True
 envlist =
     py3{4,5,6,7,8}-test-{api,sdk,example-app,ext-wsgi,ext-http-requests,ext-jaeger}
     pypy3-test-{api,sdk,example-app,ext-wsgi,ext-http-requests,ext-jaeger}
+    py3{4,5,6,7,8}-coverage
```

Comment:
will this double execute all tests? Since under the hood it's calling pytest --cov, which IIUC will run all the test again. 

If it is, I think we should eliminate duplicate tests before merging.

Reply:
> will this double execute all tests?

Yes, it will.

> If it is, I think we should eliminate duplicate tests before merging.

I don't think so currently.

There are reasons why:
 - As far as I know, we don't want to move back to using `editable mode` for the main test flow
   Please see #122 and [#208/files#r332781829](https://github.com/open-telemetry/opentelemetry-python/pull/208/files#r332781829).
 - Non-editable mode forces us to provide [fixes](https://docs.codecov.io/docs/fixing-paths) for Codecov.
 - There was (and probably *is*) a **missing files issue** on the Codecov side when using `fixes`, so it looks reasonable to use `editable mode` to avoid the issue.
   You can find some details in [this thread](https://github.com/open-telemetry/opentelemetry-python/pull/128#discussion_r322189303).
 - [Pretty ugly paths](https://github.com/open-telemetry/opentelemetry-python/pull/128#discussion_r323009457) in the [coverage report](https://user-images.githubusercontent.com/5607572/64511392-e2c22580-d2ec-11e9-9c71-c325b6569bb8.png).
 - It [was suggested](https://github.com/open-telemetry/opentelemetry-python/pull/128#discussion_r322187874) to make the coverage a different environment.

cc @Oberon00, @c24t

Reply:
I worry that this will double the build times in CI and locally for those running tox. 

I don't want to block this PR if this is the direction people want to go, but I think we should go back and find a way to not double-execute tests. Can you file a ticket once this is merged so we remember we should go back and hopefully resolve this?

Reply:
Sure, I'll submit a corresponding issue once this PR is merged.